knowledge_repo/repositories/folder.py

```diff
@@ -137,18 +144,12 @@ def _kp_get_revisions(self, path):  # slow
         raise NotImplementedError()
 
     def _kp_write_ref(self, path, reference, data, uuid=None, revision=None):
-        path = os.path.join(self.path, path)
-        if os.path.isdir(path):
-            ref_path = os.path.join(path, reference)
-            ref_dir = os.path.dirname(ref_path)
-            if not os.path.exists(ref_dir):
-                os.makedirs(ref_dir)
-            with open(ref_path, 'wb') as f:
-                return f.write(data)
-        else:
-            kp = KnowledgePost.from_file(path, format='kp')
```

Comment:
Don't delete this! This adds support for the new standalone KnowledgePost format (a compressed zip of the knowledge post folder that can be shared around).